ENIGMAsystem/SHELL/Universal_System/gif_format.cpp

```diff
@@ -19,29 +19,29 @@
 #include "gif_format.h"
```

Comment:
Did you just format this file without changing it? All that does is gunk up the blame layer.